### DISCUSSION TITLE
Is it possible to add a custom built PHP formula to brew?

### DISCUSSION THREAD

**Discussion by dmitrykuzmenkov** [Homebrew/discussions#3653] [Writing Formulae/Casks]
Hi there

We are shipping some PHP scripts that require specific extensions to build. To make it easy for the user, we want to build custom PHP as our executor with all extensions and a single command to install from the repository.

Will this formula be allowed to be added to brew or not?

I really appreciate any help you can provide.

**Comment by carlocab** [comment]
This seems like a better fit for an external tap. Here are some docs to help you get started:

https://docs.brew.sh/Taps
https://docs.brew.sh/Interesting-Taps-and-Forks
https://docs.brew.sh/How-to-Create-and-Maintain-a-Tap
https://brew.sh/2020/11/18/homebrew-tap-with-bottles-uploaded-to-github-releases/

You may wish to use this tap as an example:

https://github.com/shivammathur/homebrew-php
